products/BellHybrid/apps/application-bell-focus-timer/presenter/FocusTimerPresenter.cpp

```diff
@@ -15,6 +15,17 @@
 namespace
 {
     static constexpr std::chrono::milliseconds betweenSessionDelayTime{5000};
+    static constexpr auto summaryWindowTimeout = std::chrono::seconds{5};
+    static constexpr auto app_focus_timer_summary =
+        "<text>Well done!  <token>$VALUE</token> focus complete.</text>"; // TODO: move to translations
```

Comment:
consider adding new text in JSON for languages - if there are no translations, leave the text empty or "-"